src/dmtcp_coordinator.cpp

```diff
@@ -473,7 +473,7 @@ DmtcpCoordinator::processBarrier(const string &barrier)
   if (currentBarrier.empty()) {
     currentBarrier = barrier;
   } else {
-    JASSERT(barrier == currentBarrier);
+    JASSERT(barrier == currentBarrier) (barrier) (currentBarrier);
```

Comment:
Just below this, you have:
```
++workersAtCurrentBarrier;
releaseBarrier(barrier);
```

I'm requesting a name change:  'releaseBarrier' -> 'releaseBarrierIfAllWorkersReady'
In fact, this name change stems from an earlier commit:
> commit 09e15ca53cfaea42cf36feb19a9e0112cb6220ae
Author: Kapil Arya <kapil.arya.17@gmail.com>
Date:   Mon Sep 9 14:19:58 2019 -0700
    Newer barrier API implementation in Coordinator.

I should have noticed at that time.  Sorry.
The name change is requested because the code used to say '--workersAtCurrentBarrier;' and one could easily imagine that 'releaseBarrier' was waiting for a countdown to zero before releasing.  That was meaningful.  But with commit 09e15ca53cfaea42cf36feb19a9e0112cb6220ae , it seemed like we would increment the worker count forever and maybe do some bookkeeping about releasing internal to 'releaseBarriers'.   That is inaccurate.  The name change to 'releaseBarrierIfAllWorkersReady' makes clear that there is no internal bookkeeping.  If all workers are ready, we rel3ease.  If some workers are not release, then this function is _side-effect free_.  The more meaningful name tells us that the function will be side-effect free or it will release, and that makes reasoning about the logic _much_ easier.